collections/ghsl-urban-delineation.yaml

```diff
@@ -61,8 +61,8 @@ ProducedResults:
         Code: 21
 
 Pricing: |
- Prices are defined based on the selected area of interest (aoi) and reference year. The minimal cost for initiating request is 100€. Check [here](ghsl-urban-delineation/PRICING.MD) for more info about pricing and restrictions.
-License: "[Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/). Attribution: Contains modified Copernicus Sentinel data (year) processed by Sentinel Hub"
+ Prices are defined based on the selected area of interest (aoi) and reference year. The minimal cost for initiating request is 100€. Check [here](https://collections.eurodatacube.com/ghsl-urban-delineation/pricing.html) for more info about pricing and restrictions.
+License: "[Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/)
```

Comment:
You forgot to add the closing double qoutes ` " `

otherwise everything is good now.

Reply:
@Dorothyrono Done, thanks!